mne/defaults.py

```diff
@@ -12,34 +12,47 @@
                exci='k', ias='k', syst='k', seeg='saddlebrown', dbs='seagreen',
                dipole='k', gof='k', bio='k', ecog='k', hbo='#AA3377', hbr='b',
                fnirs_cw_amplitude='k', fnirs_fd_ac_amplitude='k',
-               fnirs_fd_phase='k', fnirs_od='k', csd='k', whitened='k'),
+               fnirs_fd_phase='k', fnirs_od='k',
+               fnirs_td_gated_amplitude='k',
+               fnirs_td_moments_amplitude='k',
+               csd='k', whitened='k'),
     si_units=dict(mag='T', grad='T/m', eeg='V', eog='V', ecg='V', emg='V',
                   misc='AU', seeg='V', dbs='V', dipole='Am', gof='GOF',
                   bio='V', ecog='V', hbo='M', hbr='M', ref_meg='T',
                   fnirs_cw_amplitude='V', fnirs_fd_ac_amplitude='V',
-                  fnirs_fd_phase='rad', fnirs_od='V', csd='V/m²',
-                  whitened='Z'),
+                  fnirs_fd_phase='rad', fnirs_od='V',
+                  # TODO: These units and scalings are wrong
+                  fnirs_td_gated_amplitude='Au',
+                  fnirs_td_moments_amplitude='Au',
+                  csd='V/m²', whitened='Z'),
     units=dict(mag='fT', grad='fT/cm', eeg='µV', eog='µV', ecg='µV', emg='µV',
                misc='AU', seeg='mV', dbs='µV', dipole='nAm', gof='GOF',
                bio='µV', ecog='µV', hbo='µM', hbr='µM', ref_meg='fT',
                fnirs_cw_amplitude='V', fnirs_fd_ac_amplitude='V',
-               fnirs_fd_phase='rad', fnirs_od='V', csd='mV/m²',
-               whitened='Z'),
+               fnirs_fd_phase='rad', fnirs_od='V',
+               fnirs_td_gated_amplitude='Au',
```

Comment:
I am not a FD expert and do not know what the correct units are. But in `mne/io/pick.py` below it is listed as `V`, but its Au here. Should they be consistent?

Reply:
Yes they should be consistent, I can change it here.

Are you happy with the new constants and channel types in general, though? If so then we can merge https://github.com/mne-tools/fiff-constants/pull/40 as it's the next step I think

Reply:
I just took at look at the fiff PR and it looks good to me.